docs/training/apis-howto/api-pytorch-ug.rst

```diff
@@ -505,6 +505,40 @@ you find that the built-in context.DataLoader() does not support your use case.
 
 See the :mod:`determined.pytorch.samplers` for details.
 
+Profiling
+---------
+
+Determined provides integrated support for the native PyTorch profiler, `torch-tb-profiler
+<https://github.com/pytorch/kineto/tree/main/tb_plugin>`_. This can be configured with
+:meth:`~determined.pytorch.PyTorchTrialContext.set_profiler` from within your Trial's ``__init__``.
+``set_profiler`` accepts the same arguments as the PyTorch plugin's ``torch.profiler.profile``
+method, but Determined will set ``on_trace_ready`` to the appropriate Tensorboard path. Stepping of
```

Comment:
split it into 2 sentences for readability and apply active voice:

``set_profiler`` accepts the same arguments as the PyTorch plugin's ``torch.profiler.profile`` method. However, Determined sets ``on_trace_ready`` to the appropriate Tensorboard path, and the stepping of the profiler during training is automatically handled.